trackers/huggingface/app.py

```diff
@@ -21,10 +21,92 @@ def add_faq() -> None:
     """
     faq = Collapsable()
     faq.add_section(
-        "Why is this so empty?",
+        "How is MLAgility different from MLPerf?",
         (
-            "Because the FAQ of huggingface website still needs to be written. "
-            "We don't use the same FAQ as in our internal dashboard."
+            "Deep learning pioneers have been judging their progress with the Machine Learning "
```

Comment:
If all these line breaks are annoying to maintain you should feel free to use a pylint-except to disable the line-length linting rule.